File: pycript/decryption.py
from .execution import execute_command
from .gethelpers import string_to_bytes, bytes_to_string

#Parameterdecrypt     -->   Parameterdecrypt 

def Parameterdecrypt(selectedlang, path, data,headers_str=None):
    body_parameter_byte = list(string_to_bytes(data))
    result = execute_command(selectedlang, path, body_parameter_byte,headers_str)
    if result is not False:
        body, header = result
        string_body = bytes_to_string(body)
        return string_body,header
    else:
        return data,headers_str

File: src/main/java/com/pycript/EncDec/Decryption.java
package com.pycript.encdec;

import org.apache.commons.lang3.tuple.Pair;
import org.apache.commons.lang3.tuple.ImmutablePair;

public class Decryption {

    public static Pair<String, String> Parameterdecrypt(String selectedLang, String decryptionPath, String body, String headersStr) {
        System.out.println("Decryption started...");
        System.out.println("Selected Language: " + selectedLang);
        System.out.println("Decryption Path: " + decryptionPath);

        String decryptedValue = "Decrypted: " + body; // Simulate decryption
        String updatedHeader = headersStr + "\r\nX-Decrypted: true"; // Add a custom header

        System.out.println("Decrypted Value: " + decryptedValue);
        System.out.println("Updated Header: " + updatedHeader);

        // Use ImmutablePair to return the values
        return new ImmutablePair<>(decryptedValue, updatedHeader);
    }
}
